src/main/java/gregtech/common/metatileentities/storage/MetaTileEntityCreativeEnergy.java

```diff
@@ -180,6 +181,7 @@ public void readFromNBT(NBTTagCompound data) {
         setTier = data.getByte("Tier");
         active = data.getBoolean("Active");
         source = data.getBoolean("Source");
+        lastEnergyIOPerSec = data.getLong("EnergyIOPerSec");
```

Comment:
This should have a check for if the nbt data contains `EnergyIOPerSec`, as I don't believe existing creative sources will have this in their nbt.